tutorials/operators/gradient_framework.ipynb

```diff
@@ -0,0 +1,1427 @@
+    "print('Hessian \\n', np.real(np.array(hessian_result)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Hessians w.r.t. State Parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 78,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "skip"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Hessian computed using the parameter shift method\n",
+      " [[-3.53553391e-01  0.00000000e+00]\n",
+      " [ 0.00000000e+00  1.38777878e-17]]\n",
+      "Hessian computed using the linear combination of unitaries method\n",
+      " [[-0.35355339  0.        ]\n",
+      " [ 0.          0.        ]]\n",
+      "Hessian computed with finite difference\n",
+      " [[-3.53591919e-01 -3.05175781e-05]\n",
+      " [-3.05175781e-05  1.52587891e-05]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "params = [a, b]\n",
+    "\n",
+    "# Get the operator object representing the Hessian\n",
+    "state_hess = Hessian(hess_method='param_shift').convert(operator=op, params=params)\n",
+    "# Assign the parameters and evaluate the Hessian\n",
+    "hessian_result = state_hess.assign_parameters(value_dict).eval()\n",
+    "print('Hessian computed using the parameter shift method\\n', np.real(np.array(hessian_result)))\n",
+    "\n",
+    "# Get the operator object representing the Hessian\n",
+    "state_hess = Hessian(hess_method='lin_comb').convert(operator=op, params=params)\n",
+    "# Assign the parameters and evaluate the Hessian\n",
+    "hessian_result = state_hess.assign_parameters(value_dict).eval()\n",
+    "print('Hessian computed using the linear combination of unitaries method\\n', np.real(np.array(hessian_result)))\n",
+    "\n",
+    "# Get the operator object representing the Hessian using finite difference\n",
+    "state_hess = Hessian(hess_method='fin_diff').convert(operator=op, params=params)\n",
+    "# Assign the parameters and evaluate the Hessian\n",
+    "hessian_result = state_hess.assign_parameters(value_dict).eval()\n",
+    "print('Hessian computed with finite difference\\n', np.real(np.array(hessian_result)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## <a name=\"qfi\"></a> QFI\n",
+    "The Quantum Fisher Information is a metric tensor which is representative for the representation capacity of a \n",
+    "parameterized quantum state $|\\psi\\left(\\theta\\right)\\rangle = V\\left(\\theta\\right)|\\psi\\rangle$ with input state $|\\psi\\rangle$, parametrized Ansatz $V\\left(\\theta\\right)$.\n",
+    "\n",
+    "The entries of the QFI for a pure state reads\n",
+    "\n",
+    "$$\n",
+    "QFI_{kl} = 4 * \\text{Re}\\left[\\langle\\partial_k\\psi|\\partial_l|psi\\rangle-\\langle\\partial_k\\psi|\\psi\\rangle\\langle\\psi|\\partial_l\\psi\\rangle \\right].$$"
```

Comment:
rhs remains `|psi\\rangle`. Need to replace it with `|\psi\\rangle`

Reply:
It's also good to move `$$` in the end of sense to the next line.

Reply:
Thanks. Took care of it.